libsolidity/inlineasm/AsmParser.cpp

```diff
@@ -468,7 +472,7 @@ assembly::Expression Parser::parseCall(Parser::ElementaryOperation&& _initialOp)
 	RecursionGuard recursionGuard(*this);
 	if (_initialOp.type() == typeid(Instruction))
 	{
-		solAssert(!m_julia, "Instructions are invalid in JULIA");
+		solAssert(m_flavour != AsmFlavour::IULIA, "Instructions are invalid in JULIA");
```

Comment:
This here should be set to `== Loose`.

Reply:
No, in strict flavour, it parses the opcode as an `assembly::Instruction` and then continues parsing the full function call and will never return the `assembly::Instruction` itself, but only an `assembly::FunctionCall` with the `solidity::Instruction` inside.